portal_spa/src/components/forms/relying-party/information.tsx

```diff
@@ -365,6 +371,34 @@ export default function RelyingPartyForm({
                 Add Attribute
               </Button>
             </div>
+            <div className="space-y-2 mt-4">
+              <FormField
+                control={control}
+                name="ready"
+                render={({ field }) => (
+                  <FormItem>
+                    <FormLabel className="flex items-center gap-2 cursor-pointer">
+                      <input
+                        type="checkbox"
```

Comment:
UI feels a bit off without this component
https://ui.shadcn.com/docs/components/checkbox